tests/testthat/_snaps/markdown.md

```diff
@@ -6,3 +6,13 @@
     
     y
 
+# markdown_body() can parse UTF-8
```

Comment:
You'll need to re-run the tests so this snapshot gets deleted.

Reply:
Yes, I forgot to commit that 😅

Reply:
Sorry, fixing again the snapshot

Reply:
Hi @hadley , now it should work